network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -72,11 +64,18 @@ <h3 class="h3-heading mb-0 mb-md-3">Minimum Security Standards</h3>
           </div>
 
           <div class="details mt-2">
-            {% include "fragments/product_criterion.html"  with  value=product.uses_encryption         help=product.uses_encryption_helptext         label="Encryption"              %}
-            {% include "fragments/product_criterion.html"  with  value=product.security_updates        help=product.security_updates_helptext        label="Security updates"        %}
-            {% include "fragments/product_criterion.html"  with  value=product.strong_password         help=product.strong_password_helptext         label="Strong password"         %}
-            {% include "fragments/product_criterion.html"  with  value=product.manage_vulnerabilities  help=product.manage_vulnerabilities_helptext  label="Manages vulnerabilities" %}
-            {% include "fragments/product_criterion.html"  with  value=product.privacy_policy          help=product.privacy_policy_helptext          label="Privacy policy"          %}
+            <div class="criterion rating">
+              {% include "fragments/product_criterion_primary_info.html"  with  label="Overall Security Rating"  value="" %}
```

Comment:
if value is empty, we should be able to just leave it off (in templates, a nonexistent var will render as an empty string)